app/_src/kubernetes-ingress-controller/guides/services/tls.md

```diff
@@ -93,7 +94,7 @@ The Ingress API supports TLS termination using the `.spec.tls` field. To termina
     spec:
      ingressClassName: kong
      tls:
-     - secretName: demo-example-com
+     - secretName: demo-example-com-cert
```

Comment:
I don't see us defining an example `Secret` with a cert that can be used in this guide. WDYT about adding it so that this guide can be following step by step (include the certificate creation)?